docs/stories/bitcoin-integration.md

```diff
@@ -0,0 +1,109 @@
+
+The effect on the custom output will be different, as the `rust-lightning` library will not ensure that the output is split among the two parties Bitcoin wallets.
+It will be the responsibility of the application built on top of `rust-lightning` to spend the custom output.
+For the 10101 wallet that will mean publishing a CET according to what the oracle attests.
+
+If you want to learn more about how a CFD protocol using DLCs works in the context of ItchySats, earlier in the year we wrote a [blog post](https://comit.network/blog/2022/01/11/cfd-protocol-explained/) about it.
+It should still be relevant to what we are building today.
+
+## Compatibility with other Lightning nodes
+
+The point of building 10101 on Lightning is to become part of the Lightning Network.
+We are excited to cater to its large and constantly growing userbase.
+To this end, it is very important for us to build a Lightning wallet which is compatible with the rest of the network.
+
+Our modifications to `rust-lightning` should just extend the capabilities of the 10101 Lightning node, without compromising its ability to route payments to and from any other node.
+As such, the 10101 wallet will only be able to do CFD trading with compatible nodes, but it should be able to pay for your coffee at your favourite cafe.
+
+## Reduced scope for the tournament
+
+Our goal for the tournament is to demonstrate that what we are building is possible and extremely cool.
+Whilst we would love to ship flawless Lightning CFD trading to your phone in less than 2 months, we have decided to cut some corners in the interest of time:
+
+- Our changes to `rust-lightning` do not need to be polished for the purpose of upstreaming.
+  We are taking these weeks as an opportunity to better understand the problem space, but we are building fast.
+  The code might not be reused in the future, but the ideas surely will.
+  If you're curious, you can follow the status of our `rust-lightning` fork [here](https://github.com/itchysats/rust-lightning/).
+- We will build ephemeral CFDs.
+  As previously mentioned, CFDs are inherently perpetual, but this would complicate matters considerably for little benefit for the purpose of the tournament.
+  We know how to build this, but it would take too much time.
+- We will not solve the problem of dual-funded channels yet.
+  This is already being [worked on by the community](https://github.com/lightning/bolts/pull/851) and it is out of scope.
+  There are ways to mitigate this problem (trusting your counterparty, using [submarine swaps](https://docs.lightning.engineering/the-lightning-network/multihop-payments/understanding-submarine-swaps)) and for the tournament we will not spend too much time on this.
+- Multi-hop custom outputs (DLCs, CFDs) are not supported.
+  Given what we said about the Lightning library not even knowing how to spend the custom output directly, it may not even make sense to support multiple hops for these types of outputs.
```

Comment:
I don't really understand this sentence - given that it is the last sentence and might need more context, maybe just change / remove / reduce it.I would either keep it more high-level or add more context so one can understand what is meant :)